Tao_Current.txt

```diff
@@ -1,73 +1,84 @@
 
-There is no membership in the IETF. Anyone may register for a meeting and then attend. The closest thing there is to being an IETF member is being on the IETF or Working Group mailing lists (see Section 2.3). This is where the best information about current IETF activities and focus can be found.
+There is no membership in the IETF. Anyone may register for a meeting and then attend. The closest thing there is to being an IETF member is being a participants on the IETF or Working Group mailing lists (see Section 2.3). This is where the best information about current IETF activities and focus can be found.
 
 Of course, no organization can be as successful as the IETF is without having some sort of structure. In the IETF's case, that structure is provided by other organizations, as described in [RFC 2028], "The Organizations Involved in the IETF Standards Process". If you participate in the IETF and read only one BCP, this is the one you should read.
 
-The IETF web site, http://www.ietf.org, is the best source for information about meetings, Working Groups, Internet-Drafts, RFCs, IETF email addresses, and much more.
+The IETF web site, [fhttps://www.ietf.org], is the best source for information about meetings, Working Groups, Internet-Drafts, RFCs, IETF email addresses, and much more.
 
 In many ways, the IETF runs on the beliefs of its participants. One of the "founding beliefs" is embodied in an early quote about the IETF from David Clark: "We reject kings, presidents and voting. We believe in rough consensus and running code". Another early quote that has become a commonly-held belief in the IETF comes from Jon Postel: "Be conservative in what you send and liberal in what you accept".
 
 The IETF is really about its participants. Because the IETF welcomes all interested individuals, IETF participants come from all over the world and from many different parts of the Internet industry. The IETF conducts its work solely in English. See Section 3.12 for information about the ways that many people fit into the IETF.
 
-One more thing that is important for newcomers: the IETF in no way "runs the Internet", despite what some people mistakenly might say. The IETF makes voluntary standards that are often adopted by Internet users, but it does not control, or even patrol, the Internet. If your interest in the IETF is because you want to be part of the overseers, you may be badly disappointed by the IETF.
+One more thing that is important for newcomers: the IETF in no way "runs the Internet", despite what some people mistakenly might say. The IETF makes voluntary standards that are often adopted by Internet users, network operators and equipment vendors, but it does not control, or even patrol, the Internet. If your interest in the IETF is because you want to be part of the overseers, you may be badly disappointed by the IETF.
```

Comment:
you probably want an oxford comma after the "operators"

Reply:
added